website/content/docs/examples/lens-app-iq.md

```diff
@@ -0,0 +1,242 @@
+---
+title: "Lens AppIQ"
+draft: false
+---
+
+# Lens AppIQ Blueprint
+
+Lens AppIQ Boundless Blueprint Tech Preview.
+
+## Lens AppIQ on Kind Cluster
+
+This blueprint bootstraps a kind cluster and installs Lens AppIQ.
+
+### Pre-req
+* [Install Boundless CLI `bctl`](https://github.com/Mirantis/boundless/blob/main/README.md#pre-requisite)
+
+### Bootsrap a Kind cluster with Lens AppIQ
+
+### Installation
+
+Set admin user's email and password.
+```shell
+export LAIQ_USER=<Admin User Email>
+export LAIQ_PASSWORD=<Admin user password>
+```
+
+```shell
+bctl apply -f lensappiq-kind-blueprint.yaml
```

Comment:
Can we add links to the Lens AppIQ blueprints? this is so that users trying it out can download those blueprints from that link?

Reply:
I added these links. This example isn't working on my machine and so was going to revisit this when I had a chance to figure out why.